packages/terra-clinical-result/src/terra-dev-site/test/clinical-result/TestResults.jsx

```diff
@@ -170,6 +170,19 @@ const mixedBPResultValue = {
   },
 };
 
+const NoDataResult = {
```

Comment:
I think a jest test for this would be appropriate given the amount of if logic in the blood pressure component

Reply:
Jest tests added https://github.com/cerner/terra-clinical/pull/614/commits/b0b5305327e8c17a49550348602d6ea79f39c368#diff-3533ae319fa17ca369ccad65829a64c6R44